base/uk.ac.stfc.isis.ibex.opis/resources/stress_rig/position.plt

```diff
@@ -1,121 +1,65 @@
-                <format_pattern>HH:mm</format_pattern>
-            </format>
-        </axis>
-    </time_axis>
+    <archive_rescale>STAGGER</archive_rescale>
     <background>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
     </background>
-    <archive_rescale>STAGGER</archive_rescale>
+    <title_font>Segoe UI|12|1</title_font>
+    <label_font>Segoe UI|9|1</label_font>
+    <scale_font>Segoe UI|9|0</scale_font>
+    <legend_font>Segoe UI|9|1</legend_font>
     <axes>
         <axis>
+            <visible>true</visible>
             <name>Position</name>
-            <font>1|Segoe UI|9.0|1|WINDOWS|1|-12|0|0|0|700|0|0|0|1|0|0|0|0|Segoe UI</font>
-            <scale_font>1|Segoe UI|9.0|0|WINDOWS|1|-12|0|0|0|400|0|0|0|1|0|0|0|0|Segoe UI</scale_font>
+            <use_axis_name>true</use_axis_name>
+            <use_trace_names>false</use_trace_names>
+            <right>false</right>
             <color>
-                <red>21</red>
-                <green>21</green>
-                <blue>196</blue>
+                <red>0</red>
+                <green>0</green>
+                <blue>0</blue>
             </color>
             <min>0.0</min>
             <max>10.0</max>
-            <log_scale>false</log_scale>
+            <grid>false</grid>
             <autoscale>true</autoscale>
-            <visible>true</visible>
-            <grid_line>
-                <show_grid_line>false</show_grid_line>
-                <dash_grid_line>true</dash_grid_line>
-                <color>
-                    <red>200</red>
-                    <green>200</green>
-                    <blue>200</blue>
-                </color>
-            </grid_line>
-            <format>
-                <auto_format>true</auto_format>
-                <time_format>false</time_format>
-                <format_pattern>############.##</format_pattern>
-            </format>
+            <log_scale>false</log_scale>
         </axis>
     </axes>
     <annotations>
     </annotations>
     <pvlist>
         <pv>
-            <name>$(PV_ROOT)POS.VAL</name>
             <display_name>Position</display_name>
             <visible>true</visible>
+            <name>$(PV_ROOT)POS.VAL</name>
             <axis>0</axis>
-            <linewidth>2</linewidth>
             <color>
-                <red>21</red>
-                <green>21</green>
-                <blue>196</blue>
+                <red>0</red>
+                <green>0</green>
+                <blue>0</blue>
             </color>
             <trace_type>AREA</trace_type>
+            <linewidth>2</linewidth>
+            <point_type>NONE</point_type>
+            <point_size>2</point_size>
             <waveform_index>0</waveform_index>
             <period>0.0</period>
             <ring_size>5000</ring_size>
             <request>OPTIMIZED</request>
-            <archive>
-                <name>RDB</name>
-                <url>jdbc:mysql://127.0.0.1/archive</url>
```

Comment:
These settings have been lost which means that the new graph is unable to load historical data.

Sorry @RichardTysonIsis I should have given you a bit more help with this. The issue here is that these graphs are "special" because they can load historical data, even when you restart the GUI. This is why it uses a log plotter widget as opposed to CSS's normal graph widget.

To get the historical data it needs to connect to an SQL database - which was being configured here. I suspect there will be an equivalent way of setting the database connection parameters under the new CSS.

The way to test this is:
- Fire up the stress rig IOC in recsim
- Put it into "Stress" control mode
- Change some setpoint values
- Check that they appear on the graph
- Close the entire IBEX GUI
- Re-open a new IBEX GUI
- Check that your previous setpoint changes appear on the graph

The other thing I noticed is that it seems to only plot on change now as opposed to continuously, the scientists prefer  continuously

Reply:
Try putting the settings into `uk.ac.stfc.isis.ibex.e4.client/plugin_customization.ini`

```
org.csstudio.trends.databrowser2/urls=jdbc:mysql://localhost/archive*jdbc:mysql://130.246.39.152/archive
org.csstudio.trends.databrowser2/use_auto_scale=true
org.csstudio.trends.databrowser2/archives=RDB|1|jdbc:mysql://localhost/archive*RDB|2|jdbc:mysql://130.246.39.152/archive
org.csstudio.trends.databrowser2/use_default_archives=true
org.csstudio.trends.databrowser2/archive_rescale=STAGGER

org.csstudio.trends.databrowser3/urls=jdbc:mysql://localhost/archive*jdbc:mysql://130.246.39.152/archive
org.csstudio.trends.databrowser3/use_auto_scale=true
org.csstudio.trends.databrowser3/archives=RDB|1|jdbc:mysql://localhost/archive*RDB|2|jdbc:mysql://130.246.39.152/archive
org.csstudio.trends.databrowser3/use_default_archives=true
org.csstudio.trends.databrowser3/archive_rescale=STAGGER
```